### ISSUE TITLE
settings.MEDIA_URL not used when saving tile data for FileListDataType

### ISSUE THREAD

**Issue by bferguso** [archesproject/arches#8960]
**Describe the bug**
The file URL prefix used by the FileListDataType is hard coded to '/files/'. If the Arches instance is behind a proxy where the URL context root isn't "/" the generated URL is invalid. If settings.MEDIA_URL is used instead of the hard-coded '/files/' prefix the generated URL would be correct based on that setting.

**To Reproduce**
Steps to reproduce the behavior (must be hosting behind a reverse proxy where / isn't the context root):
1. Add a file to any file-list node and upload the file
2. Hover over the download link
3. Note that the URL context root for the download link doesn't match the context URL of the application

**Screenshots**
![media_url](https://user-images.githubusercontent.com/6750642/189229719-aa2b7a90-6678-4b2e-8e63-88eee96d0cab.jpg)

**Expected behavior**
The settings.MEDIA_URL value should be the prefix for file download/view URL

**Your Arches Information**

*   Version used: dev/6.2.x
*   Operating System and version (desktop or mobile): OSX
*   Browser Name and version: Firefox Developer 105
*   Link to your Arches Install (optional):

**Additional context**

**Ticket Background**
*   Found by: @bferguso

**Comment by chiatt** [comment]
Fixed in #8961

**Comment by mradamcox** [comment]
I ran into an issue where this change actually seems to have broken JSON import for resources if you have MEDIA_URL set to something that is not the default. I wrote more details about it here: https://community.archesproject.org/t/issues-with-media-url-when-using-s3/1893/4.

Given that these changes presumably fixed something else that was broken they probably shouldn't be reverted, but I just wanted to make you aware of them. I could make a separate ticket about resource loading if you want.

**Comment by mradamcox** [comment]
Update: The issue I encountered was a combination of a few things, but ultimately came down to the fact that I had set `MEDIA_URL` to point to a bucket, which 1) was necessary many years ago with Arches, then 2) became an unused setting once the `/files/<file uuid>` url was added to Arches, and _then_ 3) became a real problem only after this change had been made.

I think that the real fix here is just to update the S3 docs to not recommend changing MEDIA_URL, unless there is another piece of this puzzle I am missing.
